main.py

```diff
@@ -23,7 +23,14 @@ def launch_browser():
 def log_in(browser, asnb_username, asnb_password):
 
     logging.info('🔑 Logging in')
-    browser.find_element_by_class_name("btn-login").click()
+#     browser.find_element_by_class_name("btn-login").click()
+    
+    aTagsInLi = browser.find_elements_by_css_selector('li a')
+    for a in aTagsInLi:
+        if "https://www.myasnb.com.my/uh/uhlogin/index?id=" in a.get_attribute('href'):
+            browser.get(a.get_attribute('href'))
+            break
+    
```

Comment:
Can you resubmit a PR with passing flake 8?
```
./main.py:27:1: W293 blank line contains whitespace
./main.py:33:1: W293 blank line contains whitespace
✗ ERR - error while running flake8
```

Reply:
you can run these command (i configured these in the `lint.sh` file):
`flake8`
`autopep8 --in-place --aggressive -v main.py`

Reply:
![image](https://user-images.githubusercontent.com/58810778/71051647-e97df580-2183-11ea-8c66-a502e50287c3.png)
Done